src/OrbitGl/CallstackThreadBar.cpp

```diff
@@ -305,18 +361,31 @@ std::string CallstackThreadBar::GetSampleTooltip(const PrimitiveAssembler& primi
     return unknown_return_text;
   }
 
-  std::string function_name = SafeGetFormattedFunctionName(*callstack, 0, -1);
-  std::string result =
-      absl::StrFormat("<b>%s</b><br/><i>Stack sample</i><br/><br/>", function_name.c_str());
+  ModuleAndFunctionNameToFormat innermost_module_and_function_name =
+      SafeGetModuleAndFunctionNames(*callstack, 0);
+  std::string innermost_formatted_function_name =
+      FormatFunctionName(innermost_module_and_function_name, -1);
+  std::string innermost_formatted_module_name =
+      FormatModuleName(innermost_module_and_function_name);
+
+  std::string result;
+  result += absl::StrFormat("<b>%s</b><br/>", innermost_formatted_function_name);
+  result += "<i>Stack sample</i><br/>";
+  result += "<br/>";
+  result += absl::StrFormat("<b>Module: </b>%s<br/>", innermost_formatted_module_name);
+  result += "<br/>";
   if (callstack->IsUnwindingError()) {
-    result += absl::StrFormat(
-        "<span style=\" color:%s;\"><b>Unwinding error:</b> the stack could not be unwound "
-        "successfully.<br/>%s</span>",
-        kErrorColorString, orbit_client_data::CallstackTypeToDescription(callstack->type()));
+    result += absl::StrFormat("<span style=\"color:%s;\">", kUnwindErrorColorString);
+    result += "<b>Unwinding error:</b> the stack could not be unwound successfully.<br/>";
+    result += orbit_client_data::CallstackTypeToDescription(callstack->type());
+    result += "</span><br/>";
+    result += "<br/>";
   }
-  result += "<br/><br/><b>Callstack:</b>" + FormatCallstackForTooltip(*callstack);
-  return result +
-         "<br/><br/><i>To select samples, click the bar & drag across multiple samples</i>";
+  result += "<b>Callstack:</b><br/>";
```

Comment:
Just to double-check: This is one less `<br/>` than before if I'm not mistaken, is this intended?

Reply:
I meant before. Sounds good, thanks!